includes/site-settings.php

```diff
@@ -1,19 +1,19 @@
 <?php
 
 if (function_exists('acf_add_local_field_group')) {
-    acf_add_local_field_group(array(
+    acf_add_local_field_group(array (
```

Comment:
there shouldn't be a space after array keyword, right?

Reply:
only functions shouldnt have space after